### PR TITLE
graphlib-backport 1.0.3

### PR DESCRIPTION
**Pull request by skupr-anaconda**
License: https://github.com/mariushelf/graphlib_backport/blob/master/LICENSE
Requirements: https://github.com/mariushelf/graphlib_backport/blob/master/pyproject.toml

Actions:
1. Apply an initial **percy** [patch](https://github.com/anaconda-distribution/percy/blob/main/percy/examples/patch/test_patch.json) and run locally a [script](https://github.com/anaconda-distribution/percy/blob/main/percy/examples/patch/updater_standalone.py) to: 
  - Remove `noarch python`
  - Skip `py<37`
  - Add flags `--no-deps --no-build-isolation` to `script`
  - Add missing `setuptools` and `wheel` to `host`
  - Fix `python` in `host` and `run`
2. Add `dev_url` and doc_url
3. Add `description`
4. Add license_family

Notes:
- `formulaic` https://github.com/AnacondaRecipes/formulaic-feedstock/blob/d669d6055b51a38bc1e0b7bcd4471e0e10cbc76a/recipe/meta.yaml#L27 requires it

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,29 +11,35 @@ source:
...
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+  skip: True  # [py<37]
 
 requirements:
   host:
     - pip
-    - python >=3.6,<4.0
-    - poetry
+    - python
+    - poetry 1.4.0
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:21]: 1.0 would be enough ? https://github.com/mariushelf/graphlib_backport/blob/73df515291003330c526d8700d0ea287c8ca04c0/pyproject.toml#L54 
we have the lowest common version 1.1.4 I think

Comment (skupr-anaconda) [reply recipe/meta.yaml:21]: we have only v1.4.0 with py311 support

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by akabanovs — APPROVED** [review]
LGTM

**Pull request merged**